vendor/openvino/repo.bzl

```diff
@@ -32,8 +32,8 @@ def openvino_workspace():
 
     http_archive(
         name = "openvino",
-        sha256 = "7b416d45765aea2cdd42edcdfc4e6ab4d7c4d37587530bf51a43b4c261e24a16",
-        strip_prefix = "dldt-2020.1",
+        sha256 = "0d641b158fff868ae7a969c02bf73462e335c15c40b42e3fe65ac6ea7cffea66",
+        strip_prefix = "openvino-2020.1",
         url = "https://github.com/opencv/dldt/archive/2020.1.zip",
```

Comment:
This is not related to the buffer binding. But the openvino zip file seems to have changed, I have to update the sha256 to be able to compile and pass CI. Please double check.

Reply:
Good catch, could you make this into a separate PR?

Reply:
Sure